.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  workflow_dispatch:
+
+jobs:
+    # Deploy service to prod
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.PROD_AWS_ACCESS_KEY_ID }}
```

Comment:
Created Jira DP-1314 for setting up a `dataplatform-github` AWS user in the prod account.